curriculum/challenges/english/01-responsive-web-design/applied-visual-design/use-the-strong-tag-to-make-text-bold.english.md

```diff
@@ -25,9 +25,8 @@ tests:
     testString: assert($('strong').length == 1);
   - text: The <code>strong</code> tag should be inside the <code>p</code> tag.
     testString: assert($('p').children('strong').length == 1);
-  - text: The <code>strong</code> tag should wrap around the words "Stanford University".
-    testString: assert($('strong').text().match(/^Stanford University$/gi));
-
+  - text: The <code>strong</code> tag should wrap around the words "Stanford University", or "Stanford University.".
```

Comment:
@AshNaz87 Let's just leave the `or "Stanford University." off, because the goal was just to change the test to be more flexible to the user accidentally adding the period at the end.  No reason to call attention to this fact.
```suggestion
  - text: The <code>strong</code> tag should wrap around the words "Stanford University".
```

Reply:
@RandellDawson Done

Reply:
FYI - Next time, you can simple click the Commit suggestion button and it will automatically apply the commit to the PR.